build/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Use a Jenkins LTS (Long Term Support) image as the base
+FROM jenkins/jenkins:lts-jdk17
```

Comment:
This should probably have a GHA workflow to build this image and push it to GHCR or whatever image repository Releng uses. The EKS cluster will have to pull this image from somewhere.